src/app/services/constraint.service.ts

```diff
@@ -115,11 +115,6 @@ export class ConstraintService {
   constructor(private treeNodeService: TreeNodeService,
               private studyService: StudyService,
               private resourceService: ResourceService) {
-    // Initialize the root inclusion and exclusion constraints in the 1st step
-    this.rootInclusionConstraint = new CombinationConstraint();
-    this.rootInclusionConstraint.isRoot = true;
-    this.rootExclusionConstraint = new CombinationConstraint();
-    this.rootExclusionConstraint.isRoot = true;
 
     // Initialize the root inclusion and exclusion constraints in the 1st step
     this.rootInclusionConstraint = new CombinationConstraint();
```

Comment:
so maybe not called rootInclusionConstraint anymore? just rootConstraint

Reply:
this is already renamed in https://github.com/thehyve/glowing-bear/pull/523 - removal of exclusion constraint, I will merge these 2 PRs together soon (I spitted them so it would be easier to review)